frontend/appflowy_flutter/packages/appflowy_editor_plugins/lib/src/divider/dotted_divided_shortcut_event.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:flutter/material.dart';
+import 'package:appflowy_editor_plugins/src/divider/dotted_divider_node_widget.dart';
+
+// insert divider into a document by typing three minuses.
+// ---
+
+ShortcutEvent insertDottedDividerEvent = ShortcutEvent(
+  key: 'DottedDivider',
+  command: "shift+underscore",
+  windowsCommand: "shift+underscore",
```

Comment:
```suggestion
  command: "_",
  windowsCommand: "_",
```

Reply:
This way of defining command did not work for me. Any suggestions?

Reply:
You need to rebase your branch on `main`, or merge main into your branch with git pull.

Your branch is outdated and uses appflowy_editor v0.1.4, whereas main is on version v0.1.9.